server/src/test/java/com/objectcomputing/checkins/services/role/role_permissions/RolePermissionsControllerTest.java

```diff
@@ -61,7 +60,7 @@ void testGetAllRolePermissions() {
         assertEquals(adminRole.getId(), actual.get(0).getRoleId());
         assertEquals(adminRole.getRole(), actual.get(0).getRole());
         assertNotNull(actual.get(0).getDescription());
-        assertEquals(19, actual.get(0).getPermissions().size());
+        assertEquals(adminPermissions.size(), actual.get(0).getPermissions().size());
```

Comment:
I changed this because having a hardcoded number here makes this test very brittle. I don't think it's necessary to have a test failing every time permissions are intentionally added or deleted.